### ISSUE TITLE
vab run --log now complains about private method for dummy clipboard when it used to work

### ISSUE THREAD

**Issue by rcqls** [vlang/v#9089]
**V version:** 
V 0.2.2 23f231e.38495da

**OS:** 
macos, macOS, 11.2.1, 20D74

**What did you do?**

 `vab run --log  --device auto --api 29  ~/vlang/ui/examples/users_resizable.v`

**What did you expect to see?**

 No failure (as it was 1 or 2 days ago)

**What did you see instead?**

```v
Notice: Android API level 29 is less than the recomended level (30).
/Users/rcqls/vlang/v/v failed with return code 1
/Users/rcqls/vlang/v/vlib/clipboard/clipboard.v:11:12: error: method `clipboard.Clipboard.set_text` is private
    9 | // copy copies `text` into the clipboard.
   10 | pub fn (mut cb Clipboard) copy(text string) bool {
   11 |     return cb.set_text(text)
      |               ~~~~~~~~~~~~~~
   12 | }
   13 |
/Users/rcqls/vlang/v/vlib/clipboard/clipboard.v:16:12: error: method `clipboard.Clipboard.get_text` is private
   14 | // paste returns current entry as a `string` from the clipboard.
   15 | pub fn (mut cb Clipboard) paste() string {
   16 |     return cb.get_text()
      |               ~~~~~~~~~~
   17 | }
   18 |
/Users/rcqls/vlang/v/vlib/clipboard/clipboard.v:21:5: error: method `clipboard.Clipboard.clear` is private
   19 | // clear_all clears the clipboard.
   20 | pub fn (mut cb Clipboard) clear_all() {
   21 |     cb.clear()
      |        ~~~~~~~
   22 | }
   23 |
/Users/rcqls/vlang/v/vlib/clipboard/clipboard.v:26:5: error: method `clipboard.Clipboard.free` is private
   24 | // destroy destroys the clipboard and free it's resources.
   25 | pub fn (mut cb Clipboard) destroy() {
   26 |     cb.free()
      |        ~~~~~~
   27 | }
   28 |
/Users/rcqls/vlang/v/vlib/clipboard/clipboard.v:31:12: error: method `clipboard.Clipboard.has_ownership` is private
   29 | // check_ownership returns `true` if the `Clipboard` has the content ownership.
   30 | pub fn (cb Clipboard) check_ownership() bool {
   31 |     return cb.has_ownership()
      |               ~~~~~~~~~~~~~~~
   32 | }
   33 |
/Users/rcqls/vlang/v/vlib/clipboard/clipboard.v:36:12: error: method `clipboard.Clipboard.check_availability` is private
   34 | // is_available returns `true` if the clipboard is available for use.
   35 | pub fn (cb &Clipboard) is_available() bool {
   36 |     return cb.check_availability()
      |               ~~~~~~~~~~~~~~~~~~~~
   37 | }
```

**Comment by Larpon** [comment]
Could it be related to #9062 ?

**Comment by rcqls** [comment]
Yep! @ned  had same comment on discord.

**Comment by rcqls** [comment]
Same error occurs on Linux as mentionned on discord.

**Comment by Larpon** [comment]
Attempted fix in #9107
